### DISCUSSION TITLE
Bard is parsing 'antlers' in code block

### DISCUSSION THREAD

**Discussion by harryjburk** [statamic/cms#5558] [Troubleshooting]
Hi, this is essentially a duplicate of [this](https://github.com/statamic/cms/issues/1783) issue, but I haven't been able to solve it with what was mentioned there.

Basically, we are inserting code blocks in Bard which contain Twig/Timber snippets. This means they sometimes use the same markup as Antlers, such as `{{ function('gravity_form', 1) }}`. This causes the code block to render empty rather than with the string as I'd expect.

According to the issue linked above, fields should use the parameter 'antlers: false' by default, but I can't find that parameter mentioned anywhere in the code (true or false). I tried adding it everywhere with anything to do with Bard, and no luck. Tried wrapping the code block in {{ noparse }}, tried adding '| noparse'... both also no luck, it just renders an empty code block rather than the string.

Any ideas? Am I a goose and missing something obvious?

**Comment by duncanmcclean** [comment]
Wrap the code field in a 'noparse' tag when you use it in your template.

Does that help?

**Reply by harryjburk** [reply]
Hi Duncan, thanks for the response, nah nothing at all renders then unfortunately :( Here's my test markup and the result in browser:

```
{{ noparse }}
	Noparse content:
	{{ content }}
{{ /noparse }}

<hr>

Regular content — should render {{ `function('gravity_form', 1) }}:
{{ content }}
```

![Screen Shot 2022-03-21 at 7 51 36 am](https://user-images.githubusercontent.com/93352534/159185463-f786a7a6-1294-4def-aa7e-5c9ca99c9ad2.png)

**Reply by harryjburk** [reply]
Ah, actually just remembered this is within a nav:collection with include_content, as it's rendering the content of child entries as sections. I think this is the issue that's complicating everything, as I've realised I can't use Bard sets here either. Here's the nav:collection markup:

```
{{ nav:collection:docs include_content="true" }}	 
	{{ children scope="child" }}
	   	{{ if is_current }}
		    {{ child:children }}
		    	<div id="section-{{ slug }}" class="my-5">
		    		<h3 class="mb-2">{{ title }}</h3>
		    		{{ content }}
		    	</div>				    			
		    {{ /child:children }}
		{{ endif }} 					    			    	
    {{ /children }}	   				   		 				    	 
{{ /nav:collection:docs }}
```

**Reply by edalzell** [reply]
can you provide the SMALLEST template code that shows this error?

**Comment by TravisHi** [comment]
This isn't closed?
![image](https://user-images.githubusercontent.com/9645205/159377426-3b701696-c9df-4954-9894-6834b4406203.png)

**Reply by harryjburk** [reply]
That's just there in order for it to show, it's the only workaround I've found to stop the function executing. But you're right I should close it :)
